docs/installation/mac.md

```diff
@@ -107,16 +107,26 @@ and choosing "Open" from the pop-up menu.
 
 7. Press "Close" to exit.
 
+## Learn about your Toolbox installation
```

Comment:
I moved this section up as I think it should be with the Install section. 

Is this ok?